{"uid":"4b4757e66a1912dae1a509f688f20b0f","children":[{"name":"Test defects","children":[{"name":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x5603d009efe3 <unknown>\n#1 0x5603cfdddd36 <unknown>\n#2 0x5603cfe1ac4d <unknown>\n#3 0x5603cfe1ad61 <unknown>\n#4 0x5603cfe566a4 <unknown>\n#5 0x5603cfe3b8ed <unknown>\n#6 0x5603cfe54232 <unknown>\n#7 0x5603cfe3b693 <unknown>\n#8 0x5603cfe0e03a <unknown>\n#9 0x5603cfe0f17e <unknown>\n#10 0x5603d0060dbd <unknown>\n#11 0x5603d0064c6c <unknown>\n#12 0x5603d006e4b0 <unknown>\n#13 0x5603d0065d63 <unknown>\n#14 0x5603d0038c35 <unknown>\n#15 0x5603d0089138 <unknown>\n#16 0x5603d00892c7 <unknown>\n#17 0x5603d0097093 <unknown>\n#18 0x7fd9f8bf7b43 <unknown>","children":[{"name":"User is not allowed to enter invalid phone number","uid":"5541f12628d0b81c","parentUid":"1a821a01c57654a08945244a17995c96","status":"broken","time":{"start":1683292668895,"stop":1683292668895,"duration":0},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[]}],"uid":"1a821a01c57654a08945244a17995c96"},{"name":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='localhost', port=49187): Max retries exceeded with url: /session/b067340fc1b386a94bfc2005b0dd1a35/window (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f75b436c910>: Failed to establish a new connection: [Errno 111] Connection refused'))","children":[{"name":"User is not allowed to enter alpha characters","uid":"915ac21d5b427430","parentUid":"e1e6919ddf77b5c19e37f93cdb74a0b0","status":"broken","time":{"start":1683292705621,"stop":1683292706599,"duration":978},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[]}],"uid":"e1e6919ddf77b5c19e37f93cdb74a0b0"},{"name":"selenium.common.exceptions.TimeoutException: Message:","children":[{"name":"User is able to login,this should fail","uid":"58602daa586fff46","parentUid":"31bb78a6a95ede3f78b260ed048910d8","status":"broken","time":{"start":1683292709685,"stop":1683292740927,"duration":31242},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[]}],"uid":"31bb78a6a95ede3f78b260ed048910d8"},{"name":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='localhost', port=59837): Max retries exceeded with url: /session/e1872a38dcd6512227cfce6703fa25fd/window (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f75b3219750>: Failed to establish a new connection: [Errno 111] Connection refused'))","children":[{"name":"User is able to change country code","uid":"b5c19e9faf1cd042","parentUid":"b37eea5acc7381980d5c36516b22a285","status":"broken","time":{"start":1683292744094,"stop":1683292745620,"duration":1526},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":["{'country': 'Afghanistan', 'code': '+93'}"]}],"uid":"b37eea5acc7381980d5c36516b22a285"},{"name":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='localhost', port=37515): Max retries exceeded with url: /session/46a86cb174e1acde8b6c399ab7f23489/window (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f75b31b9fc0>: Failed to establish a new connection: [Errno 111] Connection refused'))","children":[{"name":"User is able to change country code","uid":"770725cee1530d8a","parentUid":"3c0278c530af646167b34cdb68d9a684","status":"broken","time":{"start":1683292749059,"stop":1683292750610,"duration":1551},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":["{'country': 'Albania', 'code': '+355'}"]}],"uid":"3c0278c530af646167b34cdb68d9a684"}],"uid":"bdbf199525818fae7a8651db9eafe741"}],"name":"categories"}